Dockerfile

```
@@ -0,0 +1,47 @@
+# Node.js 22 (LTS) ベースイメージ
+FROM node:22-bookworm
+
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)